wiki/Options/Offset_Wizard/en.md

```diff
@@ -1,41 +1,21 @@
 
-You can use the slider given to adjust the universal offset, however, this requires you to know what the timing needs to be.
-Instead, you can use `Offset Wizard` for a more graphical representation of ticks.
+The Offset Wizard is used to assist the user in calibrating the [universal offset](/wiki/Universal_offset) to the correct value for their device. This is necessary when the [hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, across every beatmap you play. For more information on how to do this, see [How to use the Offset Wizard](/wiki/Guides%2FHow_to_Use_the_Offset_Wizard).
 
-Unlike the local song offset (specific to that difficulty only), this offset affects *every* beatmap you play.
-If you received an online offset, you can manually adjust it yourself for the difficulty you are playing using `+` or `-` to increase or decrease, respectively, the beatmap's offset by 5 milliseconds.
-You can also hold `Alt` while pressing `+` or `-` will allow you to change the offset by 1 millisecond.
+Upon opening the Offset Wizard, the user is greeted with a metronome playing and vertical green bars scrolling across the screen repeatedly. The song that was last playing/selected will begin playing, however, this can be changed at the top right of the screen via the `Now Playing` menu.
 
----
-
-To access the offset wizard, open the Options sidebar and type `offset`.
-Then click on the `Offset wizard` button.
-
-By default, the current song selected will be used.
-
-This displays your current universal audio offset in terms of milliseconds.
-
-Enabling `Half tick` will double the time for the next tick.
-If `Half tick` is enabled, the option for `Tick on the other beat` will appear.
-Enabling `Tick on the other beat` will quadruple the time for the next tick.
-Likewise, enabling `Double tick` will halven the time for the next tick.
-
-The object appeating is a "tick".
-When they pass through a middle vertical line, it will flash and will play a tick sound to represent the beat.
+The current universal offset will be displayed at the top left of the screen, which can be changed by pressing the `Up arrow` and `Down arrow`, or by using the mouse scroll wheel.
 
-Upon exiting, the value listed from this screen will be used as your universal offset.
+Underneath the universal offset display are three options that change the speed of the metronome. Enabling `Half tick` will double the amount of time between ticks on the metronome, making it slower, and reveals the `Tick on other beat` option. Enabling `Tick on other beat` will keep the time between metronome ticks the same as `Half tick`, but changes what tick it begins on, therefore offsetting it. Lastly, enabling `Double tick` will halve the amount of time between ticks on the metronome, making it faster. These options are used to adjust the metronome, to make it better fit the song that is playing.
```

Comment:
```suggestion
Underneath the universal offset value display are three options that change the speed of the metronome. Enabling `Half tick` will double the amount of time between ticks on the metronome, making it slower. This will also reveal the `Tick on other beat` option. Enabling `Tick on other beat` will keep the time between metronome ticks the same as on `Half tick`, but on a different beat. Lastly, enabling `Double tick` will halve the amount of time between ticks on the metronome, making it faster. These options are used to adjust the metronome, to make it fit the song that is playing better.
```

Reply:
The rest of this is fine, but moving "better" makes it a bit confusing to read and is kind of iffy grammar.